### PR TITLE
3DMLoader: Update

### PR DESCRIPTION
**Pull request by fraguada**
- [x] Lib: Updated to rhino3dm.js to 0.13.0 to support new functionality [4977875]

- [x] Objects: Support for SubD objects via their subdivided control net [4977875]

- [x] Objects: Light.diffuse color = THREE.Light.color [5d98e7c]

- [x] Docs: Update docs [5c84d54]

- [x] Docs: Update screenshot [e8c865f]

### CODE REVIEW

File: examples/jsm/loaders/3DMLoader.js [hunk 1]
```diff
@@ -509,8 +517,21 @@ Rhino3dmLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 				var width = ctx.measureText( geometry.text ).width + 10;
 				var height = geometry.fontHeight + 10;
 
-				ctx.canvas.width = width;
-				ctx.canvas.height = height;
+				// ref https://bl.ocks.org/duhaime/60c7083009bbd49ce50a58c371d8c818
+				var dpr = window.devicePixelRatio || 1;
+				var bsr = ctx.webkitBackingStorePixelRatio ||
```
Comment (mrdoob) [inline examples/jsm/loaders/3DMLoader.js:522]: Hmm, what is this for?

Comment (fraguada) [reply examples/jsm/loaders/3DMLoader.js:522]: See https://discourse.threejs.org/t/simplify-and-improve-code-which-uses-canvastexture/18553

Comment (fraguada) [reply examples/jsm/loaders/3DMLoader.js]: Updated that code to only use devicePixelRatio. I would appreciate simplifying all of that while still getting decent quality textures.

Comment (mrdoob) [reply examples/jsm/loaders/3DMLoader.js]: Thanks!

**Comment by mrdoob** [comment]
Thanks!

**Comment by fraguada** [comment]
PD: This PR sponsored by [McNeel](https://www.mcneel.com/).

**Pull request merged**